homeassistant/components/humidifier/reproduce_state.py

```diff
@@ -32,10 +32,9 @@ async def _async_reproduce_states(
         _LOGGER.warning("Unable to find entity %s", state.entity_id)
         return
 
-    async def call_service(service: str, keys: Iterable, data=None):
+    async def call_service(service: str, keys: Iterable) -> None:
```

Comment:
```suggestion
    async def call_service(service: str, keys: Iterable[str]) -> None:
```
I like to fully type generics while add it. We have only enabled `disallow_any_generics` for a few files, but when we want to expand that - this would be something to fix.